### PR TITLE
add horovod helm chart support

### PR DESCRIPTION
**Pull request by cheyang**
Hello,

I'm the author of [horovod helm chart](https://github.com/kubernetes/charts/tree/master/stable/horovod).  The horovod helm chart can deploy official horovod docker image with following features:

1. support official horovod docker image : uber/horovod:0.12.1-tf1.8.0-py3.5
2. support both host network and overlay network
3. customize the ssh port for  MPI connection without changing the docker image.

For more details, please read the [README](https://github.com/kubernetes/charts/blob/master/stable/horovod/README.md). I wish you can try this solution and give feedback to me. Thanks.

### CODE REVIEW

**Comment by alsrgv** [comment]
@cheyang, thanks for the PR!  Could you sign the CLA?

Also, it would be great to update reference image to `uber/horovod:0.13.3-tf1.8.0-torch0.4.0-py3.5` since it supports both TensorFlow and PyTorch.

cc @sblotner to review the wording

**Comment by cheyang** [comment]
Thank you, @alsrgv. I will update the reference image to uber/horovod:0.13.3-tf1.8.0-torch0.4.0-py3.5. But I'm not able to sign CLA, sorry. Thank you for understanding.

File: README.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ $ mpirun -np 16 \
 
 3. To run in Docker, see the [Horovod in Docker](docs/docker.md) page.
 
-4. To run in Kubernetes, see [Kubeflow](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/openmpi/README.md).
+4. To run in Kubernetes, see [Kubeflow](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/openmpi/README.md). Besides this, you can also try [Helm Chart](https://github.com/kubernetes/charts/tree/master/stable/horovod) solution.
```
Comment (sblotner) [inline README.md]: Besides this, you can also try [Helm Chart] --> You can also try the [Helm Chart]

**Comment by alsrgv** [comment]
@cheyang, sorry about that.  I will close this PR and make a separate one with a link.

**Comment by cheyang** [comment]
Thank you.

**Pull request closed without merging**